### PR TITLE
Store discontinued message instead of a flag.

### PR DESCRIPTION
**Pull request by isoos**
First part of #1055 - changes the model to store a message. We'll still need to figure where to put it and how to display it (e.g. is it a markdown-formatted text)?

### CODE REVIEW

File: app/bin/tools/discontinue.dart [hunk 1]
```diff
@@ -67,7 +67,7 @@ Future _set(String packageName, bool value) {
     if (p == null) {
       throw new Exception('Package $packageName does not exist.');
     }
-    p.isDiscontinued = value;
+    p.discontinuedMessage = 'Package is discontinued.';
```
Comment (kevmoo) [inline app/bin/tools/discontinue.dart]: Perhaps a TODO here about allowing one to provide the message?

Comment (isoos) [reply app/bin/tools/discontinue.dart]: Fixed with taking the rest of the arguments as a message.

**Comment by isoos** [comment]
@jakobr-google: shall we scrap this?

**Comment by jakobr-google** [comment]
Let's. And then revisit it later with a proper self-service solution for package authors.

**Pull request closed without merging**